tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TaskMapper.java

```diff
@@ -134,10 +140,30 @@ private TesResources createTesResources(PipelineRun run){
                 .build();
     }
 
-    private List<TesExecutor> createListExecutor(PipelineRun run){
+    private List<TesExecutor> createListExecutor(PipelineRun run) {
         return ListUtils.emptyIfNull(Arrays.asList(TesExecutor.builder()
                 .command(ListUtils.emptyIfNull(Arrays.asList(run.getActualCmd().split(SEPARATOR))))
                 .env(run.getEnvVars())
                 .build()));
     }
+
+    private List<TesTaskLog> createTesTaskLog(final Long runId) {
+        List<RunLog> runLogList = ListUtils.emptyIfNull(cloudPipelineAPIClient.getRunLog(runId));
+        List<TesExecutorLog> tesExecutorLogList = new ArrayList<TesExecutorLog>();
+        tesExecutorLogList.add(TesExecutorLog.builder()
+                .stdout(runLogList.stream().map(RunLog::getDate).findAny().toString() +
```

Comment:
something strange happening here. 
```
runLogList.stream().map(RunLog::getDate).findAny().toString() +
runLogList.stream().map(RunLog::getTaskName).iterator().next() +
runLogList.stream().map(RunLog::getLogText).iterator().next()
```
this code will produce something like:
`11.09.2019 14:53 11.09.2019 14:54 11.09.2019 14:55 Console Console Console Log text 1 Log text 2 Log text 3`

when you need:
```
11.09.2019 14:53 Console Log text 1
11.09.2019 14:54 Console Log text 2
11.09.2019 14:55 Console Log text 3
```